tests/System.IO.Compression.Tests/BrotliPerfomanceTests.cs

```diff
@@ -153,5 +154,42 @@ public void Compress(CompressionType type)
                 File.Delete(filePath);
             }
         }
+
+        [Benchmark(InnerIterationCount = Iter)]
+        [InlineData(CompressionType.CryptoRandom)]
+        [InlineData(CompressionType.RepeatedSegments)]
+        [InlineData(CompressionType.NormalData)]
+        public void Decompress(CompressionType type)
+        {
+            string testFilePath = CreateCompressedFile(type);
+            int bufferSize = 1000000;
+            byte[] data = File.ReadAllBytes(testFilePath);
+            var bytes = new byte[bufferSize];
+            foreach (var iteration in Benchmark.Iterations)
+                using (iteration.StartMeasurement())
+                    for (int i = 0; i < Benchmark.InnerIterationCount; i++)
+                    {
```

Comment:
Yeah, brackets all the way down works as well. As long as it's consistent within the nest.